R/learner_glmnet_surv_glmnet.R

```diff
@@ -10,7 +10,18 @@
 #' - `family` is set to `"cox"` and cannot be changed.
 #'
 #' @details
-#' Caution: This learner is different to learners calling [glmnet::cv.glmnet()]
+#' This learner returns two prediction types:
```

Comment:
if you have the same documentation twice you can also use a template or something like the `@inherit` / `@inheritSection` tags from roxygen

Reply:
--> this requires this information to be used to its own section

Reply:
They are a bit different between the two learners, but maybe you know a trick or two on how to do it (to discuss today)